lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -267,7 +267,8 @@ def create_database(
             raise PostgreSQLCreateDatabaseError() from e
 
         # Enable preset extensions
-        self.enable_disable_extensions(dict.fromkeys(plugins, True), database)
+        if plugins:
+            self.enable_disable_extensions(dict.fromkeys(plugins, True), database)
```

Comment:
The Pgbouncer user may not have access to the database after creating it, so the connection can be rejected here. Since PGB doesn't set any extensions, we can skip.